arm/variants/max3263x_fthr/variant.h

```diff
@@ -145,24 +155,38 @@ static const uint8_t SCK2  = PIN_SPI2_SCK;
 static const uint8_t SDA = PIN_WIRE_SDA;
 static const uint8_t SCL = PIN_WIRE_SCL;
 
-// Wire0
+// Wire0 (??)
```

Comment:
Update comment:
// Wire0: Not connected (I2CM0)

Reply:
Took your comment. Originally i was unsure if this port was connected.